### ISSUE TITLE
Stemming function

### ISSUE THREAD

**Issue by suvis705** [erikao1998/crispy-rotary-phone#17]
Add stem search into the code

**Comment by suvis705** [comment]
I'm having some problems with the stem search function. It's now it's own function in my branch, but it still just works like the exact match search.

**Comment by panukap** [comment]
Do you have a version with the original corpus? It's a little difficult to see which parts are yours, which are @erikao1998 's and which are from the original code but don't work here because of syntax errors etc.

**Comment by suvis705** [comment]
@panukap Yeah I have tried with the original corpus since it didn't work with the movie corpus. Mainly my code is the new function def stem_search(query_string, number), but of course I had to add these in the beginning:  
from nltk.stem.snowball import SnowballStemmer
from nltk.tokenize import tokenize

**Comment by panukap** [comment]
Your current code seems to still be using the search_article() function instead of stem_search(), which is probably why it works like the exact match search (ie. it is that). The stem_search() function still doesn't take into account the input words, however, and I see a couple of variables (ranked_scores_and_doc_ids, stem_words) that aren't specified before used in the function, so it might also need some changes there before it shows any results.

**Comment by suvis705** [comment]
Well yeah, the funny thing is that at least the variable ranked_scores_and_doc_ids has been used in the multi search function the same way it is in the stem search, so I don't think that is the problem. I'm now running out of time but maybe we can fix this some other time.

**Comment by panukap** [comment]
_Conclusion: Based on some test data, we got the stemming function to work, but using our movie subtitle corpus made stemming too slow to even try out (when it finally worked we got a MemoryError). Thus we decided as a team to leave the Python code there just in case, but disable searching with stemming in the HTML._
